README.md

```diff
@@ -1,12 +1,7 @@
-# Pytket Extensions
-
-This repository contains the pytket-stim extension, using Quantinuum's
-[pytket](https://cqcl.github.io/tket/pytket/api/index.html) quantum SDK.
-
 # pytket-stim
 
 [Pytket](https://cqcl.github.io/tket/pytket/api/index.html) is a python module for interfacing
```

Comment:
Given that we are using the https://tket.quantinuum.com/api-docs/ link above, it would make sense to update this as well I think.

Reply:
done